src/Resources/doc/clients.md

```diff
@@ -49,6 +49,20 @@ csa_guzzle:
             # ...
 ```
 
+You can also ```extend``` another client.
+
+```yml
+csa_guzzle:
+    clients:
+        parent_one:
+            config: 
+                 headers: 
+                     Accept: application/json
+        my_client:
+            extends: parent_one
+            # ...
+```
+
```

Comment:
Yes, it would be nice to be able to handle all these cases using the same tag, and an additional property inside the tag.